pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>org.vafer</groupId>
     <artifactId>jdependency</artifactId>
     <name>jdependency</name>
-    <version>1.4</version>
+    <version>1.5</version>
```

Comment:
Did you run a check for binary compatibility?
Either way - this is a big change that I feel deserves the version number 2.0.

Reply:
I tested on JRE 1.8, it's binary compatible. With new version requires 1.8.

I think 1.5 is enough, because only new API is added, but no change on old API. It's up to you anyway.